src/world.rs

```diff
@@ -101,6 +101,12 @@ impl World {
 
         tracing::info!(target: TRACING_TARGET, hostname = ?self.dns.reverse(addr), ?addr, "New");
 
+        // Handles connections within a host
+        self.topology.register(addr, addr);
```

Comment:
What's this one for?

Reply:
Actually, I don't think we don't. I'll remove it. If we need it in the future we can always add it.